### ISSUE TITLE
LICEcap 1.32 is not working on macOS Sonoma

### ISSUE THREAD

**Issue by jkwuc89** [justinfrankel/licecap#130]
When I attempt to record a screencast, I am unable to click through the LICECap window onto the window or the part of the screen I want to record. The clicks are ignored.

**Comment by ghost** [comment]
Yes, I have the same problem too and I'm glad that you reported it, @jkwuc89

**Comment by IvyCodes** [comment]
Same here! I've tried the solution suggested [here](https://github.com/justinfrankel/licecap/issues/31#issuecomment-893386974) with no success.

**Comment by ghost** [comment]
Yeah @IvyCodes , I just did what was suggested in the link you gave me and I can confirm on video that it doesn't work anymore. 

Video Proof
https://github.com/justinfrankel/licecap/assets/74458502/8f2e7a99-5179-4068-8646-5f1df71dd1d7

**Comment by esaruoho** [comment]
same situation with having upgraded from macOS Ventura to macOS Sonoma - clicking inside the .gif recording dialog no longer works. problematic since i need Licecap to work to do work.

**Comment by leehary** [comment]
Hi everyone, I have a solution for this on MACOS SONOMA
When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. 
It worked for me and I hope it will work for you too  XD.
If it worked for you, give me a feedback \m/

**Comment by khairulanas** [comment]
> Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/

thanks, but it doesn't work when using stage manager.

**Comment by leehary** [comment]
> > Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/
> 
> thanks, but it doesn't work when using stage manager.

haha, I've turned off it so far.

**Comment by jessepangea** [comment]
> Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/

This worked. Start the recording, minimize the LICEcap widow, do the stuff you want recorded, restore the LICEcap window and stop the recording.

**Comment by ghost** [comment]
> Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/

Your workaround does me good. Thank you very much!

**Comment by wensonsmith** [comment]
Same here, Is this project still being maintained?

**Comment by zaiMoe** [comment]
+1

**Comment by gass-git** [comment]
> Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/

Great trick! 

But this still needs a fix. Not the best UX 😅

**Comment by henriquekieckbusch** [comment]
I think it doesn't work if you have Stage Manager enabled

**Comment by jayhawx** [comment]
I'm having this issue plus I am getting a black gif with my cursor moving around and echoing... Its not showing the image at all. I removed and re-added LiceCAP to the Screen Recording System Preferences setting and its good again, but when I try to record on a secondary display, it goes black as it did before, so I have to remove and re-add for it to work... Frustrating. When it does work, the workaround that leehary shared above does work.

**Comment by ghost** [comment]
> I'm having this issue plus I am getting a black gif with my cursor moving around and echoing... Its not showing the image at all. I removed and re-added LiceCAP to the Screen Recording System Preferences setting and its good again, but when I try to record on a secondary display, it goes black as it did before, so I have to remove and re-add for it to work... Frustrating. When it does work, the workaround that leehary shared above does work.

Are you trying to screen-record a part of a coding tutorial? If so that's not a bug, that is a counter measure that some websites use to stop people from copying paid teaching content.

**Comment by jayhawx** [comment]
> > I'm having this issue plus I am getting a black gif with my cursor moving around and echoing... Its not showing the image at all. I removed and re-added LiceCAP to the Screen Recording System Preferences setting and its good again, but when I try to record on a secondary display, it goes black as it did before, so I have to remove and re-add for it to work... Frustrating. When it does work, the workaround that leehary shared above does work.
> 
> Are you trying to screen-record a part of a coding tutorial? If so that's not a bug, that is a counter measure that some websites use to stop people from copying paid teaching content.

Ah, I didn't know that! That explains so much. I will use it in my web editing software instead. Thanks!

**Comment by jayhawx** [comment]
> > I'm having this issue plus I am getting a black gif with my cursor moving around and echoing... Its not showing the image at all. I removed and re-added LiceCAP to the Screen Recording System Preferences setting and its good again, but when I try to record on a secondary display, it goes black as it did before, so I have to remove and re-add for it to work... Frustrating. When it does work, the workaround that leehary shared above does work.
> 
> Are you trying to screen-record a part of a coding tutorial? If so that's not a bug, that is a counter measure that some websites use to stop people from copying paid teaching content.

That actually isn't the problem... Still not working all applications closed. Black background even on my desktop with only LiceCAP running.... I have 3 external displays running, tried a recording on each and the MacBook display and just black with my cursor moving around...

**Comment by kianting** [comment]
> Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/

hi you suggestion worked 
![fixSCIMPaginationBug](https://github.com/justinfrankel/licecap/assets/18540972/f0087704-3c92-40dd-ac46-d5d454264f72)

As for the other issue about not being able to capture in other monitors I just resolve to capture stuff in the original mac monitor, this is a temporary work around until a bug fix arrive.

**Comment by jayhawx** [comment]
> > Hi everyone, I have a solution for this on MACOS SONOMA When I click on Start Recording from the LiceCap window, then I minimize the Licecap window and after finishing record I will open the Licecap from the taskbar and click on STOP. It worked for me and I hope it will work for you too XD. If it worked for you, give me a feedback \m/
> 
> hi you suggestion worked ![fixSCIMPaginationBug](https://user-images.githubusercontent.com/18540972/276029729-f0087704-3c92-40dd-ac46-d5d454264f72.gif) [ ![fixSCIMPaginationBug](https://user-images.githubusercontent.com/18540972/276029729-f0087704-3c92-40dd-ac46-d5d454264f72.gif) ](https://user-images.githubusercontent.com/18540972/276029729-f0087704-3c92-40dd-ac46-d5d454264f72.gif) [ ](https://user-images.githubusercontent.com/18540972/276029729-f0087704-3c92-40dd-ac46-d5d454264f72.gif)
> 
> As for the other issue about not being able to capture in other monitors I just resolve to capture stuff in the original mac monitor, this is a temporary work around until a bug fix arrive.

This doesnt work for me.

**Comment by spirosrap** [comment]
The workaround doesn't work for me either.  I see the app was developed in objective C, perhaps there's an easy way to fix this, if I come up with a solution, I will share it. 

The app is fantastic, it creates the lightest possible gif, way better than the competition. 

Edit: You correctly capture inside the window if you minimize it using "Ctrl+H". It's not working by minimizing  with clicking on the orange button.

**Comment by leehary** [comment]
Hi guys, since I upgraded to Sonoma 14.1, Licecap now working normally. :D

**Comment by wmontgomery** [comment]
> Hi guys, since I upgraded to Sonoma 14.1, Licecap now working normally. :D

Can confirm. Just updated to Sonoma 14.1 and things are working again.

**Comment by esaruoho** [comment]
i can confirm too, macOS Sonoma 14.1 and regular LICECap (v13.1) works as expected. interesting stuff.

**Comment by ronhks** [comment]
with Sonoma 14.2 it doesn't work, again :/

**Comment by whiskerside** [comment]
For Sonoma 14.2, I've tried to reinstall the app and enabled the accessibility that works now.

**Comment by whiskerside** [comment]
> For Sonoma 14.2, I've tried to reinstall the app and enabled the accessibility that works now.

I think just re-enabling the accessibility would be ok.

**Comment by HaydenOrz** [comment]
> I think just re-enabling the accessibility would be ok.

Open macOS System Settings > Privacy & Security > Screen & Audio Recording. Disable licecap and then re-enable.

I tried to do that and the problem was solved.

**Comment by frumbert** [comment]
14.4.1 (23E224), none of these suggestions working for me.

**Comment by ottocoster** [comment]
Sonoma 14.5 here. Works okay for the main laptop screen after enabling the permissions, not for my second monitor. There it's just a black screen with a moving mouse pointer.

**Comment by sahiltalwar88** [comment]
[@eclecticpassions figured out a fix](https://github.com/justinfrankel/licecap/issues/134#issuecomment-2017143989) that finally worked for me!

**Comment by jeffery9** [comment]
#132 #134
